src/WP_CLI/CommandWithMeta.php

```diff
@@ -79,13 +80,50 @@ public function list_( $args, $assoc_args ) {
 					"{$this->meta_type}_id" => $object_id,
 					'meta_key'              => $key,
 					'meta_value'            => $item_value,
-					);
-
+				);
 			}
 
 		}
 
-		print_r( $items );
+		$order   = Utils\get_flag_value( $assoc_args, 'order' );
+		$orderby = Utils\get_flag_value( $assoc_args, 'orderby' );
+
+		// Sort array.
```

Comment:
The sorting algorithm should be refactored to DRY it up. Instead of 4 functions, you should only use 1 function.

Investigate: 
```
usort( $items, function ( $a, $b ) use ( $orderby, $order ) {
    // $orderby and $order available in here as well now.
    // You can now use something like `$a->$orderby`.
}
```

Reply:
Ohh thanks @schlessera  I was searching for this kind of logic.